menustuff.json

```diff
@@ -2835,8 +2835,8 @@
   },
   "22301": {
     "OriginalText": "一時倉庫にアイテムが入っているため\nアクセスできません。\n一時倉庫を空にする必要があります。",
-    "Text": "",
-    "Enabled": false
+    "Text": "Because of items in Temp Storage,\naccess is denied.\nTemp Storage must be empty.",
```

Comment:
I think I had something like that, but I think that grammar/language is still fine like this, and it's shorter.

But if 'there are' is really better, sure let's do it.

Also, maybe I would change 'access is denied' to just 'can't access', I thank that sounds better. There was a similar string later on, and I changed that, but forgot about this one. I'll keep it in mind.